scripts/download_IMG_files.py

```diff
@@ -0,0 +1,63 @@
+"""
+This script downloads IMG files from the specified S3 bucket to the local directory in NERSC.
+"""
+
+
+import os
+from collections import defaultdict
+from pathlib import Path
+
+import boto3
+
+SECRET_KEY = os.environ.get('SECRET_KEY')
+ACCESS_KEY = 'cdm-admin'
+# Tunnel to the MinIO server - `ssh -f -N -L localhost:49002:ci07:9002 <kbase_dev_username>@login1.berkeley.kbase.us`
+ENDPOINT_URL = 'http://localhost:49002/'
+BUCKET = 'dts-staging'
+
+COLL_ROOT = Path('/global/cfs/cdirs/kbase/collections')
+SOURCE_DIR = COLL_ROOT / 'collectionssource' / 'NONE' / 'CDM' / 'IMG'
+
+FILE_SUFFIXES = ['.fna']  # ['.fna', '.gff', '.faa'] - '.gff' is also available to download. Potentially '.faa' files as well.
+
+
+def main():
+    s3_client = boto3.client('s3',
+                             endpoint_url=ENDPOINT_URL,
+                             aws_access_key_id=ACCESS_KEY,
+                             aws_secret_access_key=SECRET_KEY)
+
+    response = s3_client.list_objects_v2(Bucket=BUCKET)
+    objs = response.get('Contents', [])
+
+    genome_count = defaultdict(int)
+    total_files_downloaded = 0
+
+    for obj in objs:
+        key = obj['Key']
+
+        # Filter to download files with specified suffixes
+        if any(key.lower().endswith(suffix.lower()) for suffix in FILE_SUFFIXES):
+            parts = key.split('/')
+            genome_folder = parts[
+                -2]  # Assuming the structure 'bucket_name/uuid/img/submissions/genome_folder/file_name'
```

Comment:
What is the actual value of genome_folder?

Reply:
just an integer looks like.

Reply:
The integer is an IMG genome ID?

Reply:
I would say so. The associated files are put into this folder. I assume it's the genome id. At least some identifier if not exactly IMG genome ID. 

<img width="1034" alt="Screenshot 2024-07-01 at 1 50 31 PM" src="https://github.com/kbaseincubator/cdm-prototype/assets/3953823/ed76cec7-c3f4-4d3e-8f09-1cff317016cc">

Reply:
Ok, can you check with Jeff to confirm and add a note to the script to clarify?

Reply:
👍 verified with Jeff that it is actually the img submission id. Updated variable names/comments.

Reply:
:+1: